### PR TITLE
Narven/perf fix

### PR DESCRIPTION
**Pull request by NaraVen**
Added Exists method to improve ADLS perf
removed caching for ADLS getProperties

### CODE REVIEW

File: blobfuse/utilities.cpp [hunk 1]
```diff
@@ -276,9 +274,6 @@ int azs_getattr(const char *path, struct stat *stbuf)
 
                 stbuf->st_uid = fuse_get_context()->uid;
                 stbuf->st_gid = fuse_get_context()->gid;
-                //stbuf->st_mtime = blob_property.last_modified;
```
Comment (vibhansa-msft) [inline blobfuse/utilities.cpp:279]: We need to preserve the correct last_modified time.

File: blobfuse/src/DfsProperties.cpp [hunk 2]
```diff
@@ -99,7 +99,8 @@ dfs_properties adls_client_ext::get_dfs_path_properties(const std::string &files
         props.owner = http->get_response_header(constants::header_ms_owner);
         props.group = http->get_response_header(constants::header_ms_group);
         props.permissions = http->get_response_header(constants::header_ms_permissions);
-        props.acl = http->get_response_header(constants::header_ms_acl);
```
Comment (vibhansa-msft) [inline blobfuse/src/DfsProperties.cpp:102]: This gives file/dir permission related info in case of dfs endpoint.

Comment (NaraVen) [reply blobfuse/src/DfsProperties.cpp:102]: acl was not returned. Only permissions. So I removed.

File: blobfuse/utilities.cpp (hunk 1 above)
Comment (NaraVen) [reply blobfuse/utilities.cpp:279]: yes below the comment we are preserving it. This is just the comment.

**Pull request merged**